app.html

```diff
@@ -5,10 +5,18 @@
   <title>Title</title>
   <link rel="stylesheet" type="text/css" href="./graph.css"  />
   <script src="https://d3js.org/d3.v5.min.js"></script>
+  <script src="latex.js"></script>
+  <!--<script type="text/javascript" src="http://latex.codecogs.com/latexit.js"></script>-->
+  <!--<script type="text/javascript"> // Can be deleted if latex not used with <p>
+  LatexIT.add('p',true);
+  </script> -->
```

Comment:
This can be deleted now?

Reply:
Not really, it can be uncommented if we want to integrate latex language into <p> tags.